src/domain/users/index.types.d.ts

```diff
@@ -80,6 +79,25 @@ type NewUserInfo = {
   phoneMetadata: PhoneMetadata | null
 }
 
+type TestAccounts = {
```

Comment:
if these are `Accounts` they probably shouldn't be under `domain/users`... or are they more accurately `TestUser`?

Also the type shouldn't be pluralised:
```
testAccount: TestAccount = {..}
testAccounts = [testAccount]
```

Reply:
they're both account + user given that they're very coupled right now and creating one will automatically create the other. I'll rename to User to be consistent.

Reply:
I'll actually keep testAccount :)